### PR TITLE
Travis build

### PR DESCRIPTION
**Pull request by Tomaszsachanowski**
Repair of automated tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,18 +11,15 @@ matrix:
       dist: xenial
       sudo: true
```
Comment (tmichela) [inline .travis.yml]: I think it should be safe now to use the default image for python 3.7, you can maybe try and see if it passes

**Comment by tmichela** [comment]
Aside from the minor comment, LGTM

File: .travis.yml (hunk 1 above)
Comment (tmichela) [reply .travis.yml]: You can also add 3.8 and remove 3.5

Comment (Tomaszsachanowski) [reply .travis.yml]: Done in: 8495b32929666e122dba71e5a3e9b7ffc0c3fcd0

**Pull request merged**